doc/md/monitoring/07-notifications.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Notifications
+---
+
+To receive notifications about schema changes in your database, you can set up notifications in Atlas Schema Monitoring.
+Notifications can be sent to Slack, Workplace, or any HTTP endpoint.
+
+#### Setting up Notifications
+
+1. Log in to your Atlas Cloud account at https://auth.atlasgo.cloud.
+
+2. Click on **Monitoring** in the top-level navigation on the left side of the screen.
+
+3. Choose the database you want to monitor and click on the **Webhooks** tab.
+
+4. Click on the **Add Webhook** button and fill in the required information.
+
+<details>
```

Comment:
5. Select the integration type

Add a tabs component for each type documenting the fields in the form and their meaning.